vcr/api/v2/registry.go

```diff
@@ -74,6 +76,10 @@ func (w *Wrapper) SearchVCs(ctx echo.Context) error {
 	if err != nil {
 		return core.InvalidInputError("failed to convert query to JSON-LD expanded form: %w", err)
 	}
+
+	documentAsJson, _ := json.MarshalIndent(document, "", " ")
```

Comment:
not sure how expensive JSON marshalling is, but you could add `logrus.IsEnabled(logrus.Debug)` to avoid it when not in debug mode (I guess 99% of the time).

And maybe add a prefix to the log message: "Expanded JSON-LD search query: ..."